api/src/main/java/javax/enterprise/inject/spi/CDI.java

```diff
@@ -120,7 +119,7 @@ private static void findAllProviders() {
      * Get the CDI BeanManager for the current context
      *
      * @return the BeanManager
-     * @throws IllegalStateException if no {@link CDIProvider} is available
+     * @throws IllegalStateException if no CDI container is available
```

Comment:
Please remove this line completely. You cannot call this method if it's not available because `CDI.current()` must call `CDIProvider.getCDI()` and so would get ISE before...